satpy/etc/composites/abi.yaml

```diff
@@ -147,3 +147,261 @@ composites:
         - name: C13
     - name: C08
     standard_name: airmass
+
+  ash:
+    ## RGB Ash recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/GOES_Ash_RGB.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C15
+        - name: C13
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C14
+        - name: C11
+    - name: C13
+    standard_name: ash
+
+  dust:
+    ## RGB Dust recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/Dust_RGB_Quick_Guide.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C15
+        - name: C13
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C14
+        - name: C11
+    - name: C13
+    standard_name: dust
+
+  cloud_phase_distinction:
+    ## Cloud Phase Distinction or Day Cloud Phase (name from NASA SPORT) or Cloud Type RGB(name from EUMETSAT)
+    ## RGB Day Cloud Phase Distinction recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/Day_Cloud_Phase_Distinction.pdf
+    ## it uses the default used in etc/enhancements/generic.yaml
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C13
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    standard_name: cloud_phase_distinction
+
+  cloud_phase_distinction_raw:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C13
+    - name: C02
+    - name: C05
+    standard_name: cloud_phase_distinction
+
+
+  water_vapors1:
+    ## Water Vapors RGB or Simple Water Vapor (name from NASA SPORT)
+    ## RGB Simple Water Vapor recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/Simple_Water_Vapor_RGB.pdf
+    ## it uses the default used in etc/enhancements/generic.yaml
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C13
+    - name: C08
+    - name: C10
+    standard_name: water_vapors1
+
+  water_vapors2:
+    ## Differential Water Vapor (name from NASA SPORT)
+    ## RGB Differential Water Vapor recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/Simple_Water_Vapor_RGB.pdf
```

Comment:
The PDF should be http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_DifferentialWaterVaporRGB_final.pdf right?

Reply:
yes I already corrected it